### PR TITLE
modify some words for synapse getting started guide[skip ci]

### PR DESCRIPTION
**Pull request by viadea**
modify some words for synapse getting started guide

### CODE REVIEW

File: docs/get-started/getting-started-azure-synapse-analytics.md [hunk 1]
```diff
@@ -5,15 +5,15 @@ nav_order: 5
 parent: Getting-Started
 ---
 # Getting started with RAPIDS Accelerator on Azure Synapse Analytics
- [Azure Synapse](https://docs.microsoft.com/en-us/azure/synapse-analytics/) is an analytics service that brings
+ [Azure Synapse Analytics](https://docs.microsoft.com/en-us/azure/synapse-analytics/) is an analytics service that brings
  together enterprise data warehousing and Big Data analytics.
 
 Synapse now offers the ability to create Apache Spark pools that use GPUs on the backend to run your Spark workloads on 
-GPUs for accelerated processing. This is called as 
+GPUs for accelerated processing. These are called as
```
Comment (gerashegalov) [inline docs/get-started/getting-started-azure-synapse-analytics.md:12]: Either 
```suggestion
GPUs for accelerated processing. These are referred to as 
```
or 
```suggestion
GPUs for accelerated processing. These are called 
```

File: docs/get-started/getting-started-azure-synapse-analytics.md [hunk 2]
```diff
@@ -5,15 +5,15 @@ nav_order: 5
...
  together enterprise data warehousing and Big Data analytics.
 
 Synapse now offers the ability to create Apache Spark pools that use GPUs on the backend to run your Spark workloads on 
-GPUs for accelerated processing. This is called as 
+GPUs for accelerated processing. These are called as 
 [GPU-accelerated Apache Spark pools](https://docs.microsoft.com/en-us/azure/synapse-analytics/spark/apache-spark-gpu-concept).
 Currently it ships with the RAPIDS Accelerator for Apache Spark version 21.10.
 
-Please follow below 2 quickstart guides:
+Please follow below quickstart guides to learn how to create and use GPU pools in Azure Synapse Analytics:
```
Comment (gerashegalov) [inline docs/get-started/getting-started-azure-synapse-analytics.md:16]:
```suggestion
Please follow the Quickstart guides below to learn how to create and use GPU pools in Azure Synapse Analytics:
```

**Comment by Niharikadutta** [comment]
LGTM, thanks @viadea !

**Review by Niharikadutta — DISMISSED** [review]

**Review by gerashegalov — APPROVED** [review]
LGTM

**Comment by gerashegalov** [comment]
build

**Review by jlowe — APPROVED** [review]

**Pull request merged**